Fixed broken link

VictorWesterlund opened the pull request:
"Link rot" has claimed another victim.

The original link returns a 404 when opened. Replacing it with a Wayback Machine permalink to the last usable snapshot of the page.

VictorWesterlund committed:
Fixed broken link

Replaced broken link with a Wayback Machine snapshot.

teoli2003 reviewed (changes requested):
I found the missing page.

VictorWesterlund committed:
Added article link

Replaced Wayback Machine snapshot link to the full article on hacks.mozilla.org, courtesy of @teoli2003

Co-authored-by: Jean-Yves Perrier <jypenator@gmail.com>
